packages/suite/src/hooks/wallet/__tests__/useSendForm.test.tsx

```diff
@@ -214,9 +214,6 @@ const actionCallback = (
 };
 
 describe('useSendForm hook', () => {
-    beforeEach(() => {
-        jest.setTimeout(30000); // action sequences takes time
```

Comment:
Maybe leaving this one to avoid flakiness. I had this test failing few times locally.

Reply:
forcepushed